### DISCUSSION TITLE
A few questions on Borg

### DISCUSSION THREAD

**Discussion by chiva** [nextcloud/all-in-one#2354] [Questions] [answered]
Hi

### 1. Progress

Would it be possible to add the `--progress` flag to `borg create` ?
For initial backups (and big incrementals), it's hard to guesstimate how long could be left without any feedback.
With `progress` we would at least know how much it has processed and also that it hasn't stalled 👍 

### 2. Backup rotation

Default is "fine", but many people would have different environments.
Some people may want to backup changes for 2 years, while others don't care for anything older than 30 days.
It can greatly depend on the importance of the data stored.
Also, this can dictate a lot the backup storage space needs, so not being this configurable, may render some options unusuable.
Can't this be made configurable?

**Comment by szaimen** [comment, marked as answer]
> Would it be possible to add the `--progress` flag to `borg create` ?

We had this already in the past but we removed it because it triggered some unexpected characters in the logs as this is rather meannt to be used on CLI IIRC.

> 2. Backup rotation

Can you open a separate feature request for this?

**Reply by qk-santi** [reply]
Yes, I'd expect it to not be perfect on the logs
